setup.py

```diff
@@ -54,15 +54,15 @@
     will be forced to wait for setuptools to compile cython, instead of doing
     a fast install from a wheel.
 
-So, we chose to speep up the common cases, at the cost of requiring an
+So, we chose to speed up the common cases, at the cost of requiring an
 explicit installation of Cython in the non common cases (1) and (3).
 """
 
 
 ###############################################################################
 # Custom distutils commands
 
-DEBUG = False # whether to compile files with -g
+DEBUG = int(os.environ.get('DEBUG', False)) # whether to compile files with -g
```

Comment:
I don't like this. If by chance the user has a `DEBUG` env variable set, it might break things. Better to use `CAPNPY_DEBUG` or similar, if we want to keep the logic